contents/blog/practical-alerts.mdx

```diff
@@ -0,0 +1,167 @@
+
+Then, we can set up an alert called "No billing events sent" that gets triggered when the transactional emails have a value less than 1. In other words, it will trigger the alert if no emails get sent.
+
+We can set the frequency to check this daily. For critical alerts like this, you can add more than one email to the notification list and ensure the entire billing team receives the alert.
+
+### 4. Monitor deploys and new features
+
+With feature flags, you can set up [canary releases](/tutorials/canary-release) to safely roll out changes and make sure your new features work well before they're released to all your users.
+
+But you still want to monitor deploys and new features to verify that they're having the intended effect and not triggering unexpected errors.
+
+![Simpson, you've got a 513](https://frinkiac.com/meme/S08E23/385084.jpg?b64lines=U0lNUFNPTiwgWU9VJ1ZFIEdPVCBBIDUxMyE=)
+![That got it](https://frinkiac.com/video/S08E23/XOUcScbq8W_7yoA6nti16cCC31A=.gif)
+
+For example, you could track error events and trigger an alert if the error rate rises above a particular threshold after a new feature launch.
+
+But there are more subtle issues that aren't necessarily bugs or errors. For example, you could launch a big redesign for the product pages on your marketing website. Or change the copy on some of your CTAs. Those won't necessarily trigger errors, but they could still have a drastic impact on engagement.
+
+So you can set up an alert that coincides with the launch of a redesigned page and verify that the number of CTA clicks should hopefully be increasing, not decreasing.
+
+### 5. Alert on changes in conversion rates
+
+To set alerts on changes in your conversion rate, you can set up an insight that tracks visits to your pricing page and signups. For example, we can create a trend insight and filter for pageviews where "Path Name" equals "/pricing". And then add a filter for our `user_signed_up` event. So this insight will display a graph with a line for the number of pageviews for the pricing page as well as a separate line for the number of users that signed up.
+
+<ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/conversions_light_efbbc10280.png"
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/conversion_graph_d4ac533adf.png"
+    alt="Conversions"
+    classes="rounded"
+/>
+
+If we want to find out when the conversion rate increases or decreases by 10%, we can set up multiple alerts. We can set up one alert for when the `user_sign_up` event is increasing by 10% and a separate alert for when the `user_sign_up` event is decreasing by 10%.
+
+<ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/mutliple_alerts_light_a45b5f0ad9.png"
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/mutliple_alerts_dark_1d37e0e9b9.png"
+    alt="Multiple alerts"
+    classes="rounded"
+/>
+
+### 6. Alert on breakdown in referral sources
```

Comment:
This is basically the same as the first one

Reply:
Right. I think the differentiation for this one was just to show that you can create alerts for trends that have a breakdown. But I'll take another look and see if there's a way to further differentiate. It's a decent example that was mentioned in the GitHub issue, but you're right it could be redundant.